src/components/ControlledNumberInputField/ControlledNumberInputField.js

```diff
@@ -16,6 +16,17 @@ const useStyles = makeStyles((theme) => ({
       padding: "6px 10px 6px 10px",
       fontSize: theme.typography.fontSize,
     },
+    "& input[type=number]": {
+      "-moz-appearance": "textfield",
+    },
+    "& input[type=number]::-webkit-outer-spin-button": {
+      "-webkit-appearance": "none",
+      margin: 0,
+    },
+    "& input[type=number]::-webkit-inner-spin-button": {
+      "-webkit-appearance": "none",
+      margin: 0,
+    },
```

Comment:
This isn't quite the solution I'm looking for, all this does is handle appearance, but we also want to limit what can be entered into the input. For example:

We don't allow negative input for height
![image](https://user-images.githubusercontent.com/23342131/231685639-3aa0cac1-56dc-48e5-ad43-b76cd73480cf.png)
  
We don't want to allow this "e" character to be used :+1: 
![image](https://user-images.githubusercontent.com/23342131/231685956-880dcc34-bd6c-4f15-86f2-98ec464a5847.png)


How might we achieve this?